src/components/Trivia.js

```diff
@@ -142,20 +144,23 @@ class Trivia extends Component {
         >
           {`Question : ${question.question}`}
         </h2>
-        { options.map((answer, key) => (
-          <button
-            type="button"
-            data-testid={ answer === question.correct_answer
-              ? correct : `wrong-answer-${key}` }
-            className={ answer === question.correct_answer
-              ? correct : `wrong-answer-${key}` }
-            onClick={ (e) => this.handleCickInAnswer(e) }
-            disabled={ timeOut }
-            key={ key }
-          >
-            { answer }
-          </button>
-        ))}
+        { options.map((answer) => {
+          if (answer !== question.correct_answer) { answerIndex += 1; }
+          return (
+            <button
+              type="button"
+              data-testid={ answer === question.correct_answer
+                ? correct : `wrong-answer-${answerIndex}` }
+              className={ answer === question.correct_answer
+                ? correct : `wrong-answer-${answerIndex}` }
+              onClick={ (e) => this.handleCickInAnswer(e) }
```

Comment:
```suggestion
              onClick={ this.handleCickInAnswer }
```
Não é necessário passar uma arrow function quando quer receber o event, apenas faça isso se for chamar chamar outra função dentro do desta arrow function Ex: (e) => { this.handleClickInAnswer(e); this.otherFunction(); }